nmosauth/auth_server/db_utils.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from six import string_types
```

Comment:
Is this service python 3 only, I can't remember. If so, can we avoid using six?

Reply:
It still supports python2 yeah, the certs would just need to be generated manually now I've changed the shebang lines of the script.